### PR TITLE
fix #60 collections duplicate menu

### LINKED ISSUE #60: Duplicate collection
**Is your feature request related to a problem? Please describe.**
sometimes you have lots of collection with too many request (for example work with different version of program)

currently you can Duplicate single request but it useful if  add Duplicate for collection too.
so we can faster duplicate collection (set of Request) and update them later for specific version of program or use case

**Describe the solution you'd like**
add `Duplicate` to 3 dot of collection that generate a duplicate collection with all request inside original Collection

### PR DESCRIPTION
**Pull request by mirzakhany**
fix #60

### CODE REVIEW

File: ui/pages/requests/controller.go [hunk 1]
```diff
@@ -757,6 +761,49 @@ func (c *Controller) viewCollection(id string) {
...
+		return
+	}
+
+	colClone := col.Clone()
+	colClone.MetaData.Name += " (copy)"
+
+	dirPath, err := c.repo.GetNewCollectionDir(colClone.MetaData.Name)
+	if err != nil {
+		fmt.Println("failed to get new collection dir", err)
+		return
```
Comment (ccoVeille) [inline ui/pages/requests/controller.go:776]: Why not returning a wrapped error by changing the method signature?

Comment (ccoVeille) [reply ui/pages/requests/controller.go:776]: K k ow it's out of the PR scope. But I'm surprised by the `fmt.Println ("whatever", err) + return pattern` I see a lot in your code

Comment (mirzakhany) [reply ui/pages/requests/controller.go:776]: you right, the issue at moment is that I don't have a structure to show user this kind of errors. something like a modal widget. 
once I fix that, I'll change this.

Comment (ccoVeille) [reply ui/pages/requests/controller.go:776]: Modal or not, I would expect the error to be returned and forwarded to one method where errors will be handled only once

**Pull request merged**